metricbeat/Makefile

```diff
@@ -1,85 +1,3 @@
-# Name can be overwritten, as Metricbeat is also a library
-BEAT_NAME?=metricbeat
-BEAT_TITLE?=Metricbeat
-SYSTEM_TESTS?=true
-TEST_ENVIRONMENT?=true
-BEATS_DOCKER_INTEGRATION_TEST_ENV?=true
-ES_BEATS?=..
+ES_BEATS ?= ..
 
-# Metricbeat can only be cross-compiled on platforms not requiring CGO.
-GOX_OS=netbsd linux windows
-GOX_FLAGS=-arch="amd64 386 arm ppc64 ppc64le"
-
-DOCS_BRANCH=$(shell grep doc-branch ../libbeat/docs/version.asciidoc | cut -c 14-)
-
-include ${ES_BEATS}/libbeat/scripts/Makefile
-
-# Collects all module dashboards
-.PHONY: kibana
-kibana:
-	@rm -rf _meta/kibana.generated
-	@mkdir -p _meta/kibana.generated
-	@-cp -pr module/*/_meta/kibana/* _meta/kibana.generated
-
-# Collects all module docs
-.PHONY: collect-docs
-collect-docs:
-	mage CollectAll
-
-# Collects all module configs
-.PHONY: configs
-configs: python-env
-	@mkdir -p _meta
-	@cp ${ES_BEATS}/metricbeat/_meta/common.yml _meta/beat.yml
-	@cat ${ES_BEATS}/metricbeat/_meta/setup.yml >> _meta/beat.yml
-	@cat ${ES_BEATS}/metricbeat/_meta/common.reference.yml > _meta/beat.reference.yml
-	@${PYTHON_ENV_EXE} ${ES_BEATS}/script/config_collector.py --beat ${BEAT_NAME} --full $(PWD) >> _meta/beat.reference.yml
-	@rm -rf modules.d
-	mage config
-	@chmod go-w modules.d/*
-	@# Enable system by default:
-	@if [ -f modules.d/system.yml.disabled ]; then mv modules.d/system.yml.disabled modules.d/system.yml; fi
-
-# Generates imports for all modules and metricsets
-.PHONY: imports
-imports:
-	@mkdir -p include
-	mage imports
-
-# Runs all collection steps and updates afterwards
-.PHONY: collect
-collect: assets collect-docs configs kibana imports
-
-# Creates a new metricset. Requires the params MODULE and METRICSET
-.PHONY: create-metricset
-create-metricset: python-env
-	@${PYTHON_ENV_EXE} ${ES_BEATS}/metricbeat/scripts/create_metricset.py --path=$(PWD) --es_beats=$(ES_BEATS) --module=$(MODULE) --metricset=$(METRICSET)
```

Comment:
No we do not, but `python-env` no longer exists in the Makefile I would prefer to keep it that way. It can still be called manually.

If we want to add it back then it will need to be added in the magefile.

Reply:
If we change this make target we need to update the developer guide https://www.elastic.co/guide/en/beats/devguide/7.7/creating-metricsets.html